src/utils/globals.ts

```diff
@@ -78,6 +78,7 @@ interface PostHogExtensions {
     rrwebPlugins?: { getRecordConsolePlugin: any; getRecordNetworkPlugin?: any }
     canActivateRepeatedly?: (survey: any) => boolean
     generateSurveys?: (posthog: PostHog) => any | undefined
+    getNextSurveyStep?: (survey: any, currentQuestionIndex: number, response: string | string[] | number | null) => any
```

Comment:
we can't move `generateSurveys` because older clients are already reading from that location...

but it might be nice to put a `surveys` on here so you'd have

```
// deprecated: still in place for older clients but you should use `surveys.generateSurveys`
generateSurveys?: (posthog: PostHog) => any | undefined
surveys: {
	generateSurveys?: (posthog: PostHog) => any | undefined,
    getNextSurveyStep?: (survey: any, currentQuestionIndex: number, response: string | string[] | number | null) => 
any
}
```

so that over time it stays obvious what surveys has put on the window

not a hard requirement but i think tidier in the long run (and i wish we'd started doing this earlier)